.devcontainer/make_env.py

```
@@ -0,0 +1,221 @@
+
+LOGGER = logging.getLogger("make_env")
+
+
+def _load_env(env_files: List[Path]) -> Dict[str, str]:
+    """Evaluate files using 'sh' and return resulting environment."""
+    with TemporaryDirectory("trtllm_make_env") as temp_dir:
+        json_path = Path(temp_dir) / 'env.json'
+        subprocess.run(
+            ("(echo set -a && cat " +
+             " ".join(shlex.quote(str(env_file)) for env_file in env_files) +
+             " && echo && echo exec /usr/bin/env python3 -c \"'import json; import os; print(json.dumps(dict(os.environ)))'\""
```

Ruff (0.12.2)

Line too long (126 > 120)

(E501)